setup.cfg

```diff
@@ -0,0 +1,50 @@
+[metadata]
+name = dbt-duckdb
+author = Josh Wills
+author-email = joshwills+dbt@gmail.com
```

Comment:
@guenp @jwills should this be `author_email`? I think `author-email` is deprecated

Reply:
https://github.com/duckdb/dbt-duckdb/pull/428